x-pack/plugins/fleet/server/routes/utils/validate_endpoint_package_policy.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import Boom from '@hapi/boom';
+
+import type { NewPackagePolicyInput } from '../../../common';
+
+export const validateEndpointPackagePolicy = (input: NewPackagePolicyInput) => {
```

Comment:
can we add an unit test suite for this?

Reply:
There are units in this PR `x-pack/plugins/fleet/server/routes/package_policy/handlers.test.ts`